TypeCobol.Test/Compiler/Parser/ResultFiles/Programs/TypeCobol/ProcedureCall-overload.rdzPGM.txt

```diff
@@ -0,0 +1,44 @@
+﻿--- Diagnostics ---
+[12,50]<27,Error,Syntax>Syntax error : Symbol ValidateDatFormatt is not referenced ([12>50] on line 55)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDatFormatt does not exists ([12>50] on line 55)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat expected parameter 3 of type BOOL (actual: Alphanumeric) ([12>50] on line 62)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat expected parameter 3 of max length 1 (actual: 8) ([12>50] on line 62)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat is missing parameter 4 of type Alphanumeric ([12>50] on line 62)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat expected parameter 2 of type BOOL (actual: Alphanumeric) ([12>50] on line 62)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat expected parameter 2 of max length 1 (actual: 8) ([12>50] on line 62)
+[12,50]<27,Error,Syntax>Syntax error : Function ValidateDateFormat only takes 2 parameters ([12>50] on line 62)
```

Comment:
Just before there is :
 Function ValidateDateFormat expected parameter 3 of type BOOL (actual: Alphanumeric) ([12>50] on line 62)
Function ValidateDateFormat is missing parameter 4 of type Alphanumeric ([12>50] on line 62)

So error messages are not consistent